src/style.css

```diff
@@ -0,0 +1,75 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+body {
+  font-weight: 300;
+  font-family: Avenir, Helvetica, sans-serif;
+}
+
+.bar {
+  font-weight: 900;
+  font-family: Avenir, sans-serif;
+  display: block;
+  width: 100%;
+  box-shadow: 0 1px 8px #3d4e611a;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+}
+
+.big__bar {

```

Comment:
If you trying to use BEM methodology, you should have `bar` block and this should have modifier `big`

Or just name it regularly like `big-bar`